src/Symfony/Component/Serializer/Tests/Encoder/XmlEncoderTest.php

```diff
@@ -41,7 +41,7 @@ public function testEncodeScalar()
         $obj = new ScalarDummy();
         $obj->xmlFoo = 'foo';
 
-        $expected = '<?xml version="1.0"?>'."\n".
+        $expected = '<?xml version="1.0" encoding="UTF-8"?>'."\n".
```

Comment:
Please don't touch those tests as they don't contain unicode data, and anyway `encoding="UTF-8"` is the default value. If it doesn't work without this change, it's a BC break.